tools/accuracy_checker/configs/openpose-cmu-tf.yml

```diff
@@ -0,0 +1,54 @@
+models:
+  - name: openpose-cmu-tf
+
+    launchers:
+      - framework: tf
+        model: public/openpose-cmu-tf/opt_graph.pb
+        adapter:
+          type: human_pose_estimation
+          part_affinity_fields_out: Mconv7_stage6_L1/Conv2D
```

Comment:
Is it not required to specify outputs explicitly?
As I remember, it has concat of this 2 layers as output.